Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM quay.io/openshift/origin-must-gather:4.13.0 as builder
 
 FROM quay.io/centos/centos:stream9
 
-RUN dnf update -y && dnf install rsync -y && dnf clean all
+RUN dnf update -y && dnf install rsync python3-pyyaml -y && dnf clean all
```

Comment:
@yazug adding a python script to do masking means we have this new dependency on the `Dockerfile`; any concern on this one?